tests/testutils.py

```diff
@@ -89,7 +89,7 @@ def autodiscover_with_cleanup(*args, **kwargs):
 ContextBehParam = Union[ContextBehStr, Tuple[ContextBehStr, Any]]
 
 
-def parametrize_context_behavior(cases: List[ContextBehParam]):
+def parametrize_context_behavior(cases: List[ContextBehParam], settings: Optional[Dict] = None):
```

Comment:
Updated this test helper so that we can pass also other settings to `override_settings`.